src/ArrayHelper.php

```diff
@@ -630,17 +630,29 @@ public static function multisort(array &$array, $key, $direction = SORT_ASC, $so
         if (empty($keys) || empty($array)) {
             return;
         }
+
+        if (is_callable($key)) {
+            $keysAux = static::getColumn($array, $key);
```

Comment:
What is "aux"?

Reply:
Sorry, it is a word in Spanish. I changed it in my last PR for "keyTemp".